xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -978,7 +979,10 @@ void CGUIWindowVideoNav::GetContextButtons(int itemNumber, CContextButtons &butt
         if (node == NODE_TYPE_ACTOR && !dir.IsAllItem(item->GetPath()) && item->m_bIsFolder)
         {
           if (StringUtils::StartsWithNoCase(m_vecItems->GetPath(), "videodb://musicvideos")) // mvids
+          {
             buttons.Add(CONTEXT_BUTTON_SET_ARTIST_THUMB, 13359);
+            buttons.Add(CONTEXT_BUTTON_INFO, 21891); // Artist information
```

Comment:
There is a context menu item `CMusicInfo` for this action, use it instead. Just adding button is the outdated approach to the context menu management.